__tests__/unit/_validate/check-package-structure.test.js

```diff
@@ -56,6 +56,13 @@ describe('Check validation', () => {
 		).resolves.toEqual();
 	});
 
+	test('Resolves when filesystem matches config ignore build files', async () => {
```

Comment:
eh?

Reply:
🤔 yeah, could probably phrase that better

Reply:
updated